### DISCUSSION TITLE
[Suggestion] Select all emails loaded by the Conversation

### DISCUSSION THREAD

**Discussion by MasterInQuestion** [tutao/tutanota#5345] [Ideas]
Thereby perform actions on them at once (without having to manually one-by-one).

**Comment by ganthern** [comment]
we're working on improving operations on multiple e-mails, this in particular seems like it's just the second part of https://github.com/tutao/tutanota/issues/6

please note that you deleted a template when you posted this issue asking you to not put feature suggestions on here.

**Reply by MasterInQuestion** [reply]
My apologies for the inconsistency.
    The report was generated in good faith delivered in the most optimal manner I deem.

    Related searches within reasonable complexity restraint have been performed prior to the post.
    And the mentioned post (#6) did not score enough to be considered a duplicate. (aside, #6 is locked to collaborators only)
